### DISCUSSION TITLE
How do you solder the USB HID interface to the PCB?

### DISCUSSION THREAD

**Discussion by ElectricDream** [thehighnibble/thehighnibble.github.io#70] [S-132 add-on for IMSAI & Cromemco replicas]
Sorry, but I've been scratching my head for some time wondering how to solder the USB-HID Keyboard Interface to the PCB ... When I look at the picture on the build notes, it seems the USB interface lays between the main PCB and a 6 pin header (with the header on top of the USB interface???). 
I don't understand how the connections are made between PCB pads and the ones on the USB interface. I feel like a total idiot ;)

**Comment by GrantMeStrength** [comment]
![IMG_0017](https://github.com/thehighnibble/thehighnibble.github.io/assets/1928855/ad20165c-2838-4809-936e-05791510d178)
![IMG_0012](https://github.com/thehighnibble/thehighnibble.github.io/assets/1928855/0401b6c6-bbb2-47b2-85ec-cf9351c57b45)
![IMG_0011](https://github.com/thehighnibble/thehighnibble.github.io/assets/1928855/eb250672-51ae-496c-92a3-51f49595daa6)

**Comment by ElectricDream** [comment]
Thanks for the pictures. I ended doing the same :)

**Comment by dmcnaugh** [comment]
There is a YouTube video specifically on this topic https://youtu.be/02tlehBJUas

**Reply by ElectricDream** [reply]
Hi Dave. Yes. I finally came across this video, but a little bit too late ;)
